README.md

```diff
@@ -73,6 +73,17 @@ Rails.logger.info "Meatball"
 - `msg` = If values given is not a hash, it is treated as `{"msg":<value>"}`
 - All duration related fields are in milliseconds
 
+To log additional fields that are available on the `ActionDispatch::Request` object, set in `config/application.rb`:
+```ruby
+Superlogger.log_extra_fields = lambda do |request|
```

Comment:
I'm wondering if there's a better name for this property.

1. "Extra" seems to have some connotation of "unnecessary/not strictly needed", which feels a little odd. 😅 
2. This method doesn't do the actual logging (which might be implied by `log` as a verb), but more configures which fields should be logged.

Ideas:
* `include_fields`
* `include_log_fields`
* `other_log_fields`
* `additional_log_fields` (this was one of the first that came to mind, but it's long)

This is non-blocking if we can't find a better name, though.

Reply:
I'm in favour of `include_log_fields` for the reasons you've stated - will make the change, thanks!